test/Configure/CheckLibWithHeader_extra_libs.py

```diff
@@ -38,138 +38,23 @@
 libA_dir = Path(test.workdir) / "libA"
 libA_dir.mkdir()
 libA = str(libA_dir / (dll_ + 'A' + _dll))  # for existence check
-
-test.write(
-    str(libA_dir / "libA.h"),
-    """\
-#ifndef _LIBA_H
-#define _LIBA_H
-
-// define BUILDINGSHAREDLIB when building libA as shared lib
-#ifdef _MSC_VER
-# ifdef BUILDINGSHAREDLIB
-#  define LIBA_DECL __declspec(dllexport)
-# else
-#  define LIBA_DECL __declspec(dllimport)
-# endif
-#endif // WIN32
-
-#ifndef LIBA_DECL
-# define LIBA_DECL
-#endif
-
-LIBA_DECL void libA(void);
-#endif // _LIBA_H
-""",
-)
-test.write(
-    str(libA_dir / "libA.c"),
-    """\
-#include <stdio.h>
-#include "libA.h"
-
-LIBA_DECL void libA(void) {
-    printf("libA\\n");
-}
-""",
-)
-test.write(
-    str(libA_dir / "SConstruct"),
-    """\
-SharedLibrary(target='A', source=['libA.c'], CPPDEFINES='BUILDINGSHAREDLIB')
-""",
-)
+test.dir_fixture(['fixture', 'checklib_extra', 'libA'], 'libA')
```

Comment:
I think you should just be able to do this once as 
`test.dir_fixture(['fixture', 'checklib_extra'])`

Reply:
I suppose. If you think it's worth doing yet another iteration...